docs/design/features/multilevel-sharedfx-lookup.md

```diff
@@ -208,10 +208,11 @@ Both files carry the filenames for dependencies that must be found. They can be
 
 At last, the coreclr is loaded into memory and called to run the application.
 
-### Hostpolicy changes for 2.1+
-For 2.0, there are several probing paths that are used to find the dependencies. These paths follow a certain order and the first assembly found wins and that location will be passed to the coreclr. For example, the local app location has priority over the shared framework locations and if the same assembly exists in both locations, the coreclr will end up using the local app's copy of that assembly.
+### Probing for Assemblies
+Assemblies are found by looking through probing paths in a certain order. The first found matching assembly wins, and that location will be passed to CoreCLR. For example:
```

Comment:
This paragraph says "first found matching assembly wins" and the 2.1+  paragraph says "the assembly with the highest version wins". Seems to be conflicting statements.

Reply:
Thanks for pointing it out, @jkotas. I've fixed this.